wled00/set.cpp

```diff
@@ -136,6 +136,18 @@ void handleSettingsSet(AsyncWebServerRequest *request, byte subPage)
       doInitBusses = true;
     }
 
+    ColorOrderMap com = {};
+    for (uint8_t s = 0; s < WLED_MAX_COLOR_ORDER_MAPPINGS; s++) {
```

Comment:
What if there are entries missing? It would be better to break instead of adding 0s.

Reply:
Fair point, thank you!